interfaces/builtin/x11.go

```diff
@@ -19,19 +19,96 @@
 
 package builtin
 
-const x11Summary = `allows interacting with the X11 server`
+import (
+	"github.com/snapcore/snapd/interfaces"
```

Comment:
Can you reorder the imports so that local ones come last? `goimports -local github.com/snapcore/snapd -w x11{,_test}.go` should do the trick.

Reply:
Done